### PR TITLE
22 set up publication

### PR DESCRIPTION
**Pull request by shilohd**

### CODE REVIEW

**Review by RuSnook — APPROVED** [review]
Spotted a few things with potential improvements in future. but nothing that would impact running. Looks good to me!

File: quarto/main/index.qmd [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## How do analysts use programming and software engineering in the UK Public Sector?
+
+Please note it is possible that the figures presented here will change in future. 
+
+While any changes to the data are not likely to be substantial, we are planning to carry out more in-depth analysis and the commentary may change. As such, we recommend linking to these pages rather than sharing this analysis any other way at this stage. We will be regularly updating these pages with additional outputs and insights. 
+
+The coding in analysis and research survey (CARS) was developed to better understand how programming is used by government analysts. Most questions focused on whether and how analysts use code in their work. The survey is also concerned with Reproducible Analytical Pipelines (RAP). You can find links to further information on RAP at the bottom of this page. 
+
+The survey was distributed to government analysts and researchers in the Autumn of 2021. The results presented here summarise the key findings from 912 respondents from over 50 public sector organisations. You can find the 2021 findings in [last year's publication](https://best-practice-and-impact.github.io/CARS-3/).
```
[the remark this answers is not in the document]
Comment (RuSnook) [reply quarto/main/index.qmd:17]: This was the only part I wanted to make sure on, @Jacobb164, do you know if this is correct?

**Comment by RuSnook** [comment]
all tests passed too.

File: quarto/main/index.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (ldavies99) [reply quarto/main/index.qmd]: Yes, this is fine.

**Pull request merged**